Source/WebKit/WebProcess/GPU/media/WebMediaStrategy.cpp

```diff
@@ -88,9 +90,19 @@ std::unique_ptr<WebCore::NowPlayingManager> WebMediaStrategy::createNowPlayingMa
     return WebCore::MediaStrategy::createNowPlayingManager();
 }
 
+bool WebMediaStrategy::hasThreadSafeMediaSupport() const
+{
+#if ENABLE(GPU_PROCESS)
+    return m_useGPUProcess && WebCore::MediaPlayer::hasThreadSafeSupport();
```

Comment:
Why isn't m_useGPUProcess sufficient? I would guess that if m_useGPUProcess is true, we are using remote media players.